src/storybook/stand-alone-documentaion/typography-new/typography.stories.mdx

```diff
@@ -1,16 +1,16 @@
 import { Meta } from "@storybook/addon-docs";
 import { ComponentName, Frame, Link, RelatedComponents } from "../../components";
-import { TextStyles } from "./text-styles/text-styles";
 import { TextColors } from "../../components/text-colors/text-colors";
+import { TextStyles } from "./text-styles/text-styles";
 import { COLORS, SHADOW, SPACING } from "../../components/related-components/component-description-map";
 import { typographyExample1, typographyExample2 } from "./assets";
 import "./typography.scss";
 
-<Meta title="Foundations/Typography" />
+<Meta title="Foundations/Typography2" />
```

Comment:
maybe just keep it as `Typography` and keep the old one as Typography (Legacy)

Reply:
just push fix